stubs/cv2-stubs/__init__.pyi

```diff
@@ -6,6 +6,7 @@ import builtins as _mod_builtins
 import cv2 as _mod_cv2
 import numpy as np
 from . import Error as Error
+from typing import overload
```

Comment:
I'd recommend using `@typing.overload` on the two overload lines for consistency with the rest of the file.

Reply:
@debonte Good point, just changed it!